index.html

```diff
@@ -1,225 +1,242 @@
+                 <button class="chevron-button">
+                 <img src="./images/chevron-right.png" alt="" class="chevron">
+                  </button>
+                </div>
             </li>
-          </ul>
-          <div class="action">
-            <button class="btn">See Project</button>
-          </div>
-        </div>
-      </div>
-    
-      <div class="section5">
-        <div class="top">
-          <h4 class="about-heading">About <br>Myself</h4>
-          <p class="about-description">Hello I'm a software developer! I can help you build a product, feature or website. Look
-            through some of my work and experience. If you like what you see and have a project you need coded, don't
-            hestiate to contact me.</p>
-          <h2 class="social order">LET'S CONNECT</h2>
-
-          <ul class="social-links">
-            <li class="li1"><a href="https://twitter.com/YoungeCjay" target="_blank"><img src="./images/Twitter icon.png"></a></li>
-            <li class="li2"><a href="https://www.linkedin.com/in/clinton-younge-83386a25a/" target="_blank"><img src="./images/Linkedin icon.png"></li>
-            <li class="li3"><a href="#"><img src="./images/Letter M.png"></a></li>
-            <li class="li4"><a href="https://github.com/ClintonYounge" target="_blank"><img src="./images/Github Kitty.png"></a></li>
-            <li class="li5"><a href="#"><img src="./images/Finger icon.png"></a></li>
-          </ul>
-
-          <button class="resume">Get my resume</button>
-        </div>
-        <div class="skills-list">
-          <div class="languages">
-            <div class="container">
-              <span class="lang">Languages</span>
-              <i class="fa-solid fa-angle-down btn1"></i>
-            </div>
-            <div class="language-list">
-              <ul>
-                <li class="l-list"><img class="images" src="./images/Javascript Icon.png"><span class="js">Javascript</span></li>
-                <li class="l-list"><img class="images" src="./images/HTML icon.png"><span class="js">HTML</span></li>
-                <li class="l-list"><img class="images" src="./images/CSS icon.png"><span class="js">CSS</span></li>
-              </ul>
+
+        <li>
+            <div class="skills">
+                <span class="skills-title">Skills</span>
+                <button class="chevron-button">
+                <img src="./images/chevron-right.png" alt="" class="chevron">
+                </button>
             </div>
-          </div>
-          <div class="container">
-            <span class="lang">Frameworks</span>
-            <i class="fa-solid fa-angle-right btn1"></i>
-          </div>
-          <div class="container">
-            <span class="lang">Skills</span>
-            <i class="fa-solid fa-angle-right btn1"></i>
-          </div>
-        </div>
-      </div>
+        </li>
+    </ul>
     </section>
 
+    <section id="contact">
```

Comment:
- [ ] Good job integrating the Formspree service successfully. One of the objectives of this project is to understand how to parse a design template precisely. 🙂️
Kindly add a `border-radius` on your inputs to give them a rounded feel around the corners. 😎️
There is an empty space on the right side of the section, kindly adjust it to match the design. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/213136639-3f3c50da-c2a4-4340-baaa-1f5aea6f955c.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/213136634-a646243c-74f0-40cd-9ed2-553f132a7295.png" width="300">|